bin/otu_to_tax_name.py

```diff
@@ -1,26 +1,40 @@
 #!/usr/bin/env python
 
-import argparse
 import sys
-from phylotoast import otu_calc as otuc, util
+import argparse
+from phylotoast import otu_calc as otuc
+importerrors = []
+try:
+    import biom
+except ImportError:
+    importerrors.append("biom")
+try:
+    import pandas as pd
+except ImportError:
+    importerrors.append("pandas")
+if len(importerrors) > 0:
+    for err in importerrors:
+        print("Please install missing module: {}".format(err))
+    sys.exit()
 
 
 def handle_program_options():
-    parser = argparse.ArgumentParser(description="Convert a list of OTU IDs to a list of\
-                                                  OTU IDs paired with Genus_species \
-                                                  identifiers and perform reverse lookup, \
-                                                  if needed.")
-    parser.add_argument("-i", "--otu_id_fp", required=True,
-                        help="Either a text file containing a list (one per line) \
-                        of OTU IDs, or a tab-separated (classic) BIOM-format file.")
-    parser.add_argument("-t", "--taxonomy_fp", required=True,
-                        help="A file associating OTU ID with a full taxonomic specifier.")
-    parser.add_argument("-o", "--output_fp", default="otu_taxonomy.txt",
-                        help="For a list input, a new file containing a list of OTU IDs \
-                              and their corresponding short taxonomic identifiers \
-                              separated by tabs. For a BIOM file input, a new \
-                              mapping file with all the OTU IDs replaced by the short\
-                              identifier.")
+    parser = argparse.ArgumentParser(description="Convert a list of OTU IDs to a list of "
+                                                 "OTU IDs paired with Genus_Species "
+                                                 "identifiers and perform reverse lookup,"
+                                                 " if needed.")
+    parser.add_argument("-i", "--otu_table", required=True,
+                        help="Input biom file format OTU table. [REQUIRED]")
+    parser.add_argument("-oid", "--otu_id_fp", required=True,
+                        help="Either a text file containing a list (one per line) "
```

Comment:
We should change the help text to indicate that we the point of this parameter is the OTU IDs. Something like: "A single or multi-column (tab-separated) file containing the OTU IDs to be converted in the first column. [REQUIRED]"